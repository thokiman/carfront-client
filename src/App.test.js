import AddCar from "./components/add-car/add-car.component";
import Modal from "./components/modal/modal.component";
import TestRenderer from "react-test-renderer";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe("<Add/>", () => {
  it("render a snapshot", () => {
    const tree = TestRenderer.create(<AddCar />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders five <TextInput/> components", () => {
    const wrapper = shallow(<AddCar />);
    expect(wrapper.find("TextField")).toHaveLength(5);
  });
});
