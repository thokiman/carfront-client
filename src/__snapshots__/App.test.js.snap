// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Add/> render a snapshot 1`] = `
<div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-text"
    disabled={false}
    id="modal-button-add"
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      New Car
    </span>
  </button>
</div>
`;
